### ISSUE TITLE
How to install gdm themes with gdm-tools

### ISSUE THREAD

**Issue by kshitijaucharmal** [realmazharhussain/gdm-tools#5]
When I install a new gdm theme from gnome-look.org, and copy it in /usr/share/themes, it doesn't show up in set-gdm-theme list, only the adwaita ones are visible. How to install gdm theme with this tool??

**Comment by realmazharhussain** [comment]
Actually, GDM themes available on gnome-look.org are not used by GDM anymore. Maybe GDM2 or earlier versions of GDM3 used them but GDM3 (the current version of GDM) doesn't use them anymore.
Now, GDM3 uses a gnome-shell theme as its theme.

Download themes from [here](https://www.gnome-look.org/browse?cat=134&order=latest).
Extract theme to `/usr/share/themes/` so that the directory structure is `/usr/share/themes/${ThemeName}/gnome-shell/` and has a file `/usr/share/themes/${ThemeName}/gnome-shell/gnome-shell.css`.

### TL;DR
1. Download and Install gnome-shell themes instead of GDM themes
2. If Gnome-Tweaks can set it as Shell theme, `set-gdm-theme` should also be able to list and set it as GDM theme.
